app/views/dashboard/_activity.html.erb

```diff
@@ -64,13 +64,13 @@
           <%= translation('dashboard._activity.dropdown.wiki') %>
         </label>
       </a>
-      <!--
-      <li class="divider"></li>
-      <li class="dropdown-header"><%=translation('tag.index.sort_by')%></li>
-      <li><a href="#"><a href="#">Post date</a></li>
-      <li><a href="#"><a href="#">Recent activity</a></li>
-      -->
-    </div>
+           <!--    
+            <li class="divider"></li>
+            <li class="dropdown-header"><%=translation('tag.index.sort_by')%></li> 
+            -->
+           <li><a href="#"><a href="#"><%= translation('dashboard._activity.post_date') %></a></li>
+           <li><a href="#"><a href="#"><%= translation('dashboard._activity.recent_activity') %></a></li>
```

Comment:
Great @Axuby! We're getting there. You still need to move the `-->` after `recent_activity` here. 

It's strange that we're making the changes inside the comment, but I guess that's what the original issue said! 🤷 

Also, if you can clean up the whitespace (extra tabs) too, that would be great.

Reply:
Ok, done @noi5e